src/types/api/auth.ts

```diff
@@ -0,0 +1 @@
+export type RedirectUrlDto = { redirectURL: string }
```

Comment:
nit: change the name to `PublicFormAuthRedirectDto`?

Reply:
np will do